test/Shared/Microsoft.DotNet.Scaffolding.Shared.Tests/CodeAnalysisHelperTests.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using Microsoft.DotNet.Scaffolding.Shared.CodeModifier;
+using Xunit;
+
+namespace Microsoft.DotNet.Scaffolding.Shared.Tests
+{
+    public class CodeAnalysisHelperTests
+    {
+        [Fact]
+        public void LoadCodeAnalysisProjectTest()
+        {
+            var files = new List<string>();
+            var projectPath = string.Empty; // TODO test project
```

Comment:
@deepchoudhery what do you think would make sense for this?

Reply:
This is good since all we want to test is the initialization of the AdhocWorkspace without any msbuild assembly exceptions. We can add one for loading documents later.